shell_completions.md

```diff
@@ -71,7 +71,7 @@ PowerShell:
 `,cmd.Root().Name()),
 	DisableFlagsInUseLine: true,
 	ValidArgs:             []string{"bash", "zsh", "fish", "powershell"},
-	Args:                  cobra.ExactValidArgs(1),
+	Args:                  MatchAll(ExactArgs(1), OnlyValidArgs),
```

Comment:
This should be `cobra.MatchAll(ExactArgs(1), OnlyValidArgs),`

Reply:
Thanks, I fixed it it is actually `cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs)` :)

Reply:
ha!  Good point!

Reply:
While we're at it, would you mind also fixing the 3 missing `cobra.` prefixes in:

https://github.com/spf13/cobra/blob/badcce14f8e828cda4d8ac404a12448700de1441/user_guide.md?plain=1#L352

Reply:
For sure, fixed and checking if there is any left.